### PR TITLE
New info risks table

### PR DESCRIPTION
**Pull request by noyaossi**
i added the modal to risks page, to the current main. please let me know if there are some problems. thank you :)

### CODE REVIEW

File: controllers/endpoints.js [hunk 1]
```diff
@@ -134,4 +134,16 @@ async function getControlPanel(req, res) {
     }
 }
 
-module.exports = { getEndpoints, addNode, getControlPanel };
\ No newline at end of file
+async function getHostScripts(req, res) {
+    try {
+        const hostId = req.params.hostId;
+        const scripts = await fleetService.getScriptsByHost(hostId);
+        console.log("Scripts fetched:", scripts);  // Debugging line
```
Comment (tal-az) [inline controllers/endpoints.js]: remove debug logging from production

File: services/fleet.js [hunk 2]
```diff
@@ -280,6 +280,24 @@ async function removeHostFromFleetById(hostId) {
...
+async function getScriptsByHost(hostId) {
+    try {
+        // Construct the URI to fetch scripts for the specific host
+        const scriptUri = `/api/v1/fleet/hosts/${hostId}/scripts`;
+        
+        // Fetch the scripts using the existing fleetApiGetRequest function
+        const response = await fleetApiGetRequest(scriptUri);
+        
+        // If the response contains scripts, return them; otherwise, return an empty array
+        console.log(`Scripts for host ${hostId}:`, response.scripts);  // Debugging line
```
Comment (tal-az) [inline services/fleet.js]: again

**Review by tal-az — APPROVED** [review]
Minor fixes!
havent ran the code but it doesnt seem to break anything 👍

File: views/risks.ejs [hunk 3]
```diff
@@ -59,10 +59,92 @@
...
+                            <td>${scriptName}</td>
+                            <td>${executionTime}</td>
+                            <td>${status}</td>
+                        </tr>
+                    `;
+                });
+                $('#scriptDetails').html(scriptDetails);
+                $('#scriptModal').modal('show');
+            } catch (error) {
+                console.error('Error fetching scripts:', error);
```
Comment (maziland) [inline views/risks.ejs]: The whole `script` tag should be in a separate `js` files.
I looks great, but it's easier to change and manage when client side javascript is in a separate file.
you should create `public/js/risks.js` and move the content there, then include it in the ejs file like so: `<script src=/js/risks.js></script>

**Review by maziland — CHANGES_REQUESTED** [review]

**Comment by noyaossi** [comment]
i created a new file in public/js/risks.js, and moved the script from views/risks.ejs.

**Review by tal-az — APPROVED** [review]
Note that in statistics.ejs best practice is formatting and crating elements will be done in client side as you've done in public/js/statistics.js. No biggie but for next time. looks good!

**Review by maziland — APPROVED** [review]

**Pull request merged**